### PR TITLE
Version 1.1.0

### PR DESCRIPTION
**Pull request by zxenia**
1. Improved ingestion error handling
2. Added CanDIG auth middleware
3. Added django-cors
4. Added filtering for phenopackets, patients, resources, experiments apps
5. Changed name to Katsu

### CODE REVIEW

File: chord_metadata_service/chord/ingest.py [hunk 1]
```diff
@@ -23,6 +23,7 @@
 __all__ = [
     "METADATA_WORKFLOWS",
     "WORKFLOWS_PATH",
+    "IngestError",
     "ingest_resource",
```
Comment (brouillette) [inline chord_metadata_service/chord/ingest.py:27]: A bit of nitpicking for consistency:
Should these also be in capital letters with underscores between words?

**Review by brouillette — COMMENTED** [review]
I have yet to use this so I'm unaware of the full scale of the changes, but on a code level, it looks good to me!

File: chord_metadata_service/chord/ingest.py (hunk 1 above)
Comment (zxenia) [reply chord_metadata_service/chord/ingest.py:27]: No, in this case, it is a list that just exposes all different variables and functions that can be imported from here. And they are of different types that follow their own style, e.g. `IngestError`  follows class style in python recommended by PEP8, `ingest_resource ` follows recommended function style.

Comment (brouillette) [reply chord_metadata_service/chord/ingest.py:27]: Got it!
Thank you -- approved

**Review by brouillette — APPROVED** [review]

**Review by brouillette — APPROVED** [review]

**Pull request merged**